packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-header/pipeline-ai.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { GenerativeAIInput } from '@mongodb-js/compass-components';
+import { connect } from 'react-redux';
+import createLoggerAndTelemetry from '@mongodb-js/compass-logging';
+
+import {
+  changeAIPromptText,
+  cancelAIPipelineGeneration,
+  runAIPipelineGeneration,
+} from '../../../modules/pipeline-builder/pipeline-ai';
+import type { RootState } from '../../../modules';
+
+const { log, mongoLogId, track } = createLoggerAndTelemetry('AI-PIPELINE-UI');
+
+const onSubmitFeedback = (feedback: 'positive' | 'negative', text: string) => {
+  log.info(mongoLogId(1_001_000_229), 'PipelineAI', 'AI pipeline feedback', {
+    feedback,
+    text,
+  });
+
+  track('PipelineAI Feedback', () => ({
+    feedback,
+    text,
+  }));
+};
+
+type PipelineAIProps = Omit<
+  React.ComponentProps<typeof GenerativeAIInput>,
+  'onSubmitFeedback'
+>;
+
+function PipelineAI(props: PipelineAIProps) {
+  return (
+    <GenerativeAIInput
```

Comment:
This component adds margin that should not be added making it misaligned with all the other content in the toolbar:

![image](https://github.com/mongodb-js/compass/assets/5036933/7bd44fbb-b0c2-4296-8cde-3a02bd931267)

Layout positioning should be handled by parents, not children, especially for reusable components.